runtime-sdk/src/module.rs

```diff
@@ -151,6 +151,22 @@ impl MigrationHandler for Tuple {
     }
 }
 
+/// Block handler.
+#[impl_for_tuples(30)]
```

Comment:
I see we use this on most traits that support automatic implementations (methods with no return values, no associated types). Do we actually need/use any of these in tuples anywhere (or plan to in future)? Or are we just doing this as a "convention" where we can?

Reply:
Oh yes, this is a neat feature that allows us to do things like:
https://github.com/oasisprotocol/oasis-bridge/blob/91db3f59a73611f482b86d12890ed8efde4d75b2/runtime/src/lib.rs#L14-L17

And then call all these methods on the `Runtime::Modules` type (which basically combines multiple modules).

Reply:
Ah yeah that's nice :+1: i was actually looking for a runtime implementation but didn't found it in this repo - didn't think about looking over at bridge repo :grin: